NUMPY/LINALG/EIG/EIG.py

```diff
@@ -1,13 +1,15 @@
 from flojoy import OrderedPair, flojoy, Matrix, Scalar
 import numpy as np
-
+from collections import namedtuple
+from typing import Literal
 
 import numpy.linalg
 
 
 @flojoy(node_type="default")
 def EIG(
     default: OrderedPair | Matrix,
+    select_return: Literal["w", "v"] = "w",
```

Comment:
It's because some of these functions return a "namedtuple" so right now instead of trying to return both I'm just letting the user choose which of the two to return. 

![image](https://github.com/flojoy-io/nodes/assets/41165864/51ef8930-9590-45aa-92b7-121c2942c289)

Reply:
In the docs they call it something different than the docstring (which is where I scrape from). w and v are from the docstring.